crates/extensions/c8y_firmware_manager/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "c8y_firmware_manager"
+description = "Thin-edge device firmware management for Cumulocity"
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+rust-version = { workspace = true }
+license = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+
+[dependencies]
+async-trait = "0.1"
+c8y_api = { path = "../../../crates/core/c8y_api" }
+c8y_http_proxy = { path = "../../extensions/c8y_http_proxy" }
+csv = "1.1"
+download = { path = "../../../crates/common/download" }
```

Comment:
Actually, it needs `Auth`, and this can be used by `tedge_api` already.

In test, it uses `download::DownloadError::FromIo`, I believe it's okay to move this dependency to `dev-dependencies`.